src/shared/Atlassian.Bitbucket.UI.Avalonia/Atlassian.Bitbucket.UI.Avalonia.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <RuntimeIdentifiers>osx-x64;linux-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>osx-x64;linux-x64;osx-arm64</RuntimeIdentifiers>
```

Comment:
Again, let's verify `osx-arm64` is a [supported RID](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#macos-rids) (ditto for all the other `.csproj` files.

Reply:
Per the [dotnet runtime repo](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json#L2117l), osx-arm64 is a supported RID. I could not verify if it was a supported RID in .NET 5, but it definitely is in .NET 6.

Reply:
Excellent!